lib/API/JSON/v1/Subscribers.php

```diff
@@ -201,6 +192,64 @@ function subscribe($data = []) {
       ]);
     }
 
+    $is_builtin_captcha_required = false;
```

Comment:
It seems we have over 100 lines of code for captcha validation. Maybe we could move it into a private method which would return a bad request in case of invalid captcha or maybe even refactor this to some separate class.
I think it would make `subscribe` method in API class much more readable.